pkg/cvo/availableupdates.go

```diff
@@ -124,6 +125,18 @@ func (optr *Operator) setAvailableUpdates(u *availableUpdates) {
 		} else {
 			klog.Warningf("Unrecognized condition %s=%s (%s: %s): cannot judge update retrieval success", u.Condition.Type, u.Condition.Status, u.Condition.Reason, u.Condition.Message)
 		}
+
+		sort.Slice(u.Updates, func(i, j int) bool {
+			vi := semver.MustParse(u.Updates[i].Version)
+			vj := semver.MustParse(u.Updates[j].Version)
+			return vi.GTE(vj)
+		})
+
+		sort.Slice(u.ConditionalUpdates, func(i, j int) bool {
+			vi := semver.MustParse(u.ConditionalUpdates[i].Release.Version)
+			vj := semver.MustParse(u.ConditionalUpdates[j].Release.Version)
+			return vi.GTE(vj)
+		})
```

Comment:
I am not sure what we are sorting here as I do not see the conditional statement here. For example here is the example I found in the internet https://www.includehelp.com/golang/sort-slice-function-with-examples.aspx

Reply:
Also can we add an unit test for this?

Reply:
I'm sorting both `Updates` (which feeds ClusterVersion's `status.availableUpdates`) and `conditionalUpdates`.  We had an existing unit test that I had to [update][1] now that we're sorting `Updates`.  I guess we could add unit coverage for the `availableUpdates` sort too, but it's basically the same sorting code, so I'm personally ok without trying to work out a `pkg/cvo/...` test harness for this portion.

[1]: https://github.com/openshift/cluster-version-operator/commit/c9dd4792f3efeb3ac7964f1f2211e2b756173304#diff-9fce41c8cfcd572d1aa9649ddd5b3644fee050fb667e6846f9e07bd7b7e70fcbL2605